packages/node_modules/@ciscospark/plugin-phone/src/call.js

```diff
@@ -808,6 +808,7 @@ const Call = SparkPlugin.extend({
 
   // The complexity in createOrJoinLocus is largely driven up by fairly readable `||`s
   /* eslint-disable complexity */
+  /* eslint-disable no-param-reassign */
```

Comment:
This was just a warning, right? let's not disable warnings, that way way when we decide to fix them, we know where there are.

Reply:
(except for warnings that we're never going to fix. For example, payload transforms rely on a bunch of things that generate warnings and that'll probably never change)

Reply:
I thought it was an error but I’ll double check